packages/ide-extension/src/enhancement/commandHandler.ts

```diff
@@ -1,40 +1,30 @@
 import { commands, window } from 'vscode';
-import type { Command, TerminalCommand, VSCodeCommand } from '@sap/guided-answers-extension-types';
+import type { Command } from '@sap/guided-answers-extension-types';
+import { isTerminalCommand, isVSCodeCommand } from '@sap/guided-answers-extension-types';
+import { logString } from '../logger/logger';
 
 /**
- * Type guard to check if command is VSCode command.
- *
- * @param exec - exec property of command
- * @returns true: is VSCode commmand; false: no VSCode command
- */
-function isVSCodeCommand(exec: VSCodeCommand | TerminalCommand): exec is VSCodeCommand {
-    return (exec as VSCodeCommand).commandId !== undefined;
-}
-
-/**
- * Type guard to check if command is terminal command.
- *
- * @param exec - exec property of command
- * @returns true: is terminal commmand; false: no terminal command
- */
-function isTerminalCommand(exec: VSCodeCommand | TerminalCommand): exec is TerminalCommand {
-    return (exec as TerminalCommand).cwd !== undefined;
-}
-
-/**
- * Execute command execution requests (actions from app info webview).
+ * Execute command execution requests.
  *
  * @param command - the command which could be VSCode command or terminal command
  */
 export function handleCommand(command: Command): void {
     if (isVSCodeCommand(command.exec)) {
+        logString(
+            `Execting VSCode command '${
+                command.exec.commandId
+            }'. Full command info including arguments:\n${JSON.stringify(command, null, 2)} `
+        );
         commands.executeCommand(command.exec.commandId, command.exec.argument);
     }
     if (isTerminalCommand(command.exec)) {
         const terminal = window.createTerminal(`Guided Answers Terminal`);
```

Comment:
Not for this pr, but since I noticed, does it makes sense to simply have the name as `Guided Answers extension by SAP` or `Guided Answers extension`

Reply:
Are you talking about the name of the terminal? I've changed it to `Guided Answers extension by SAP`